### PR TITLE
Replace existing navigation with axis navbar component - WIP

### PR DESCRIPTION
**Pull request by acolytec3**
WIP PR to replace existing site navigation with axis navbar componet as part of [developer.centrifuge.io - issue #129](https://github.com/centrifuge/developer.centrifuge.io/issues/129)

Not ready for review

### CODE REVIEW

File: src/components/Navigation/index.js [hunk 1]
```diff
@@ -1,436 +0,0 @@
...
-    href={href}
-    {...rest}
-  >
-    {children}
-  </ExternalLink>
-)
-
-const BrandLink = () => (
-  <div
-    onContextMenu={e => {
```
Comment (lucasvo) [inline src/components/Navigation/index.js]: This feature of the navbar needs to be persisted. Can you add the capability to bind the contextmenu to a specific action to the standard component?

File: src/components/Navigation/index.js [hunk 2]
```diff
@@ -1,436 +0,0 @@
...
-    /* Dropdown Item Link Styles */
-    > li > a {
-      line-height: 1rem;
-      display: block;
-      padding: 0.5rem 1rem;
-    }
-  }
-`
-
-const Nav = styled(Box)`
```
Comment (lucasvo) [inline src/components/Navigation/index.js]: When removing those styling changes, the reference of the design should be the current website not what is configured in axis.

Comment (acolytec3) [reply src/components/Navigation/index.js]: On this point, I'm assuming I need to completely rework the component to incorporate a dropdown for secondary menu items along the lines of the current site, right?

**Comment by acolytec3** [comment]
Confirmed as no longer needed.

**Pull request closed without merging**